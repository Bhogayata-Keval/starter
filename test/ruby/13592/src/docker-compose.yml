# Generated by Cloud66 Starter
---

custom_web:
  build: .

worker:
  build: .

