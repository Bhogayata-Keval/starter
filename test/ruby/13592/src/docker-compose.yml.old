---

custom_web:
  build: .
  
  ports:
    
 	   
  - "3000:3000"
    	
    
  
  	
  links:
	
  - mysql
  - mysql:mysql.cloud66.local
	
  environment:
  	
  - 