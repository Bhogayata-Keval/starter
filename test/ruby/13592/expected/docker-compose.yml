custom_web:
  build: .
  links:
  	- postgresql
  	- redis

worker:
  build: .
  links:
  	- postgresql
  	- redis

postgresql:
  image: postgresql

redis:
  image: redis
